core/src/main/java/tachyon/worker/eviction/EvictStrategy.java

```diff
@@ -28,7 +28,51 @@
  * when actually evicting blocks, some blocks may be not allowed to be evicted, it may result in
  * having to try more than one time to get enough space.
  */
-public interface EvictStrategy {
+public abstract class EvictStrategy {
+  /**
+   * Actions to take when the new block is added
+   */
+  public void onAdd(long blockId) {}
+
+  /**
+   * Actions to take when delete the block
+   */
+  public void onDelete(long blockId) {}
```

Comment:
omg, the eclipse format checker won't adjust this automatically, do u guys maintain the alphabetical order manually ?